### PR TITLE
[s]Adds major ommision to bots

### PR DESCRIPTION
**Pull request by Tastyfish**
## butts
To build:
1. Groin organ + unsecured voice analyzer
2. (Pen to rename)
3. Add cyborg arm.
4. butt

### CODE REVIEW

**Comment by Alex6511** [comment]
:+1:

File: code/modules/mob/living/simple_animal/bot/buttbot.dm [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+	..()
+	spawn(3)
+		var/datum/job/clown/J = new/datum/job/clown
+		access_card.access += J.get_access()
+		prev_access = access_card.access
+
+/mob/living/simple_animal/bot/buttbot/Destroy()
+	if(superfart_timer)
+		deltimer(superfart_timer)
+		superfart_timer = null
```
Comment (Fox-McCloud) [inline code/modules/mob/living/simple_animal/bot/buttbot.dm]: `return ..()`

**Comment by Fox-McCloud** [comment]
:+1:

**Comment by TheDZD** [comment]
You horrible, horrible person.

File: code/modules/mob/living/simple_animal/bot/buttbot.dm (hunk 1 above)
Comment (Tastyfish) [inline code/modules/mob/living/simple_animal/bot/buttbot.dm]: Derp

**Comment by Spacemanspark** [comment]
:-1: We don't need more butt humor.

**Comment by ProperPants** [comment]
I can feel the memes creeping into my blood like a cancer.
:+1: :-1: :confused: :expressionless:

**Comment by TheDZD** [comment]
@Spacemanspark @ProperPants Check today's/yesterday's/tomorrow's date. :P

**Comment by Fox-McCloud** [comment]
@Spacemanspark 

Butt out of the conversation, will ya?

**Comment by tigercat2000** [comment]
if this is reverted after april first :+1: :x: :100:
if it's kept, i fucking quit :smile_cat:

**Comment by kylejen** [comment]
butt the butt butt the butt?

**Comment by Tastyfish** [comment]
Yes, this is for April 1st. :stuck_out_tongue:

**Comment by ProperPants** [comment]
Looks like I was the butt of the joke. Fucking April 1st...

**Comment by kylejen** [comment]
Looks butt I was butt butt of the butt. Butting April butt.

**Comment by Iamgoofball** [comment]
i'm reopening this PR for reals

**Comment by tigercat2000** [comment]
@Iamgoofball no

**Comment by Fox-McCloud** [comment]
@tigercat2000 yes

**Comment by Spacemanspark** [comment]
@Fox-McCloud no

**Comment by Tastyfish** [comment]
Maybe

**Comment by Tastyfish** [comment]
no ifs, ands, or butts

**Pull request closed without merging**